cms/templates/admin/cms/grouper/change_form.html

```diff
@@ -0,0 +1,21 @@
+{% extends "admin/change_form.html" %}{% load i18n %}
+{% block field_sets %}
+  {% block language-tabs %}
+    {% if language_tabs %}
+      <div id="page_form_lang_tabs" class="js-language-tabs" data-message="{{ changed_message|escape }}">
+          {% for lang_code, lang_name in language_tabs %}
```

Comment:
Currently, the change for only supports a language filter. The reason is simple: For languages, we know all possible values (since they are configured in the settings). For any other grouping field, we do not know in general. They need a separate section in the template. 

My thinking would be that eventually the change form would support all grouping fields (incl. none). For now, it does this, as long as they are `language` ;-).  If you had any other grouping field, you'd have to provide your own change form (which is pretty common for Djano, right?).

I'd like to give the other grouping fields a go, however, in a **subsequent PR**. It's hard to test since so far there are no examples for other grouping fields.

Reply:
Sounds logical and sane to me.